playpen/ansible/roles/dev/files/bashrc

```diff
@@ -82,6 +82,16 @@ pclean() {
     done
     popd
 
+    # Scrub out the Squid cache
+    if [ -d /var/spool/squid ]; then
+        sudo systemctl stop squid
+        sudo rm -rf /var/spool/squid
+        sudo mkdir --context=system_u:object_r:squid_cache_t:s0 --mode=750 /var/spool/squid
+        sudo chown squid:squid /var/spool/squid
+        sudo squid -z
```

Comment:
I think it's strange that squid has this flag. If you don't run it, does starting squid fail? I don't mind it being here, just remarking that it's weird to me.